CVSROOT:	/cvs/mesa
Module name:	Mesa
Repository:	Mesa/src/glx/x11/
Changes by:	idr@gabe.	05/08/01 09:30:24

Log message:
  Fix build problems related to finding Xthreads.h.

Modified files:
      Mesa/src/glx/x11/:
        glxclient.h glxext.c 
  
  Revision      Changes    Path
  1.12          +1 -1      Mesa/src/glx/x11/glxclient.h
  http://cvs.freedesktop.org/mesa/Mesa/src/glx/x11/glxclient.h
  1.15          +0 -5      Mesa/src/glx/x11/glxext.c
  http://cvs.freedesktop.org/mesa/Mesa/src/glx/x11/glxext.c

Index: src/glx/x11/glxclient.h
===================================================================
RCS file: /cvs/mesa/Mesa/src/glx/x11/glxclient.h,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -b -B -r1.11 -r1.12
--- Mesa-6.3.1.1/src/glx/x11/glxclient.h	29 Jul 2005 17:30:18 -0000	1.11
+++ Mesa-6.3.1.1/src/glx/x11/glxclient.h	1 Aug 2005 16:30:24 -0000	1.12
@@ -61,7 +61,7 @@
 #include "glapitable.h"
 #include "glxextensions.h"
 #if defined( XTHREADS )
-# include "Xthreads.h"
+# include <X11/Xthreads.h>
 #elif defined( PTHREADS )
 # include <pthread.h>
 #endif
Index: src/glx/x11/glxext.c
===================================================================
RCS file: /cvs/mesa/Mesa/src/glx/x11/glxext.c,v
retrieving revision 1.14
retrieving revision 1.15
diff -u -b -B -r1.14 -r1.15
--- Mesa-6.3.1.1/src/glx/x11/glxext.c	26 Jul 2005 22:53:38 -0000	1.14
+++ Mesa-6.3.1.1/src/glx/x11/glxext.c	1 Aug 2005 16:30:24 -0000	1.15
@@ -51,11 +51,6 @@
 #include <assert.h>
 #include "indirect_init.h"
 #include "glapi.h"
-#ifdef XTHREADS
-# include "Xthreads.h"
-#elif defined(PTHREADS)
-# include <pthread.h>
-#endif
 #include "glxextensions.h"
 #include "glcontextmodes.h"
 #include "glheader.h"
